packages/xo-web/src/xo-app/tasks/index.js

```diff
@@ -277,6 +277,10 @@ export default class Tasks extends Component {
     )
   )
 
+  _itemsPerPageContainer = () => this.itemsPerPage
+
+  _setItemsPerPageContainer = element => (this.itemsPerPage = element)
```

Comment:
- [The first call will be with `null`]( https://reactjs.org/docs/refs-and-the-dom.html#caveats-with-callback-refs). Do we really want to assign `null` to `itemsPerPage`? 
- Why not save it in the state?